components/TransactionDetails/TransactionDetails.styled.js

```diff
@@ -1,15 +1,15 @@
 const { default: styled } = require("styled-components");
 
 export const DetailsContainer = styled.div`
-display: ${(props) => props.$showModal ? "none" : "block"}; `
-
+  display: ${(props) => (props.$visible ? "none" : "block")};
```

Comment:
this is very unintutive, if I pass "true" to the prop $visible the element is hidden. Please revert this such that it is visible on $visible being "true"